sys/ztimer/Kconfig

```diff
@@ -77,7 +77,7 @@ choice
     default ZTIMER_SEC_BACKEND_RTC if !BOARD_NATIVE && \
                                       !CPU_COMMON_SAM0 && \
                                       !CPU_COMMON_EFM32 && \
-                                      !CPU_FAM_F1
+                                      !CPU_STM32
```

Comment:
I don't think we need these, now ztimer_sec is modelled in Makefile.dep. Native should be kept though AFAIU.